analyzers/src/SonarAnalyzer.Common/Helpers/KnownType.cs

```diff
@@ -451,6 +452,7 @@ internal sealed partial class KnownType
         internal static readonly KnownType System_SystemException = new("System.SystemException");
         internal static readonly KnownType System_Text_RegularExpressions_Regex = new("System.Text.RegularExpressions.Regex");
         internal static readonly KnownType System_Text_RegularExpressions_RegexOptions = new("System.Text.RegularExpressions.RegexOptions");
+        internal static readonly KnownType System_Text_RegularExpressions_GeneratedRegexAttribute = new("System.Text.RegularExpressions.GeneratedRegexAttribute");
```

Comment:
It will be used in those to come. But it can be dropped here indeed. When using this attribute, the compiler will also tell you that your regex is invalid.